spec/kitchen/driver/openstack_spec.rb

```diff
@@ -211,28 +212,6 @@
       end
     end
 
-    context 'when executed with a bourne shell' do
```

Comment:
Yeah i tried to get them to pass; but something about the rspec foo and me didnt click. If you want to give em a shot, I'm all about that.